index.css

```
@@ -710,3 +719,184 @@ body a > .commit-hash {
+    background: var(--background-color);
+    user-select: none;
+    outline: none;
+    font: inherit;
+}
+
+.theme-card.snow {
+    overflow: hidden;
+    background-clip: padding-box;
+    --incorrect-color: var(--gray-2);
+    --box-border-color: var(--gray-3);
+    background:
```

Biome (2.4.6)

[error] Unexpected shorthand property background after background-clip

(lint/suspicious/noShorthandPropertyOverrides)